storage/store/consumer_progress.go

```diff
@@ -0,0 +1,87 @@
+package store
```

Comment:
This is actually renamed from `storage/badger/consumer_progress.go`, github wasn't able to recognize this move, but think it's a new file.